src/entitysdk/downloaders/ion_channel_model.py

```diff
@@ -0,0 +1,22 @@
+"""Download functions for IonChannelModel entities."""
+
+from pathlib import Path
+
+
+def download_one_mechanism(client, ic, mechanisms_dir="./mechanisms"):
```

Comment:
mechanism_dir -> output_dir
also I would rename it into `download_ion_channel_mechanism`
ic -> ion_channel_model
Remove default

Reply:
done in latest commit